doc/rbd/iscsi-target-cli.rst

```diff
@@ -0,0 +1,166 @@
+=============================================================
+Configuring the iSCSI Target using the Command Line Interface
+=============================================================
+
+The Ceph iSCSI gateway is the iSCSI target node and also a Ceph client
+node. The Ceph iSCSI gateway can be a standalone node or be colocated on
+a Ceph Object Store Disk (OSD) node. Completing the following steps will
+install, and configure the Ceph iSCSI gateway for basic operation.
+
+**Requirements:**
+
+-  A running Ceph Luminous or later storage cluster
+
+-  CentOS 7.4, Linux kernel v4.12 or newer
```

Comment:
Hey,

CentOS 7.4, when it is released, will work with the kernel it is released with.

I am working on support for other distros like ubuntu and for them, you will need a 4.13 or newer kernel.